src/Weasel.Postgresql/Tables/IndexDefinition.cs

```diff
@@ -436,20 +452,47 @@ private static string canonicizeColumn(string expression)
             return CanonicizeCast(expression);
         }
 
-        private static string removeSortOrderFromExpression(string expression, out SortOrder order)
+        private static string removeSortOrderFromExpression(string expression, out SortOrder order, out NullsSortOrder? nullsSort)
```

Comment:
**SUGGESTION:** Maybe we could return tuple here instead of multiple `out` params? As it's a private method then I think it would be more explicit and won't mess with public API.

Reply:
Fixed and handled this suggestion.